package.json

```diff
@@ -106,12 +108,18 @@
   "scripts": {
     "build": "gatsby build",
     "develop": "gatsby develop",
-    "format": "prettier --write \"**/*.{js,jsx,ts,tsx,json,md}\"",
+    "format": "prettier --write \"*/**/*.{js,jsx,ts,tsx,json,css,scss}\"",
     "start": "netlify dev",
     "serve": "gatsby serve",
     "clean": "gatsby clean",
     "test": "echo \"Write tests! -> https://gatsby.dev/unit-testing\" && exit 1",
-    "validate": "node bin/validate.js"
+    "validate": "node bin/validate.js",
+    "prepare": "husky install"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "prettier --write \"*/**/*.{js,jsx,ts,tsx,json,css,scss}\""
```

Comment:
same here as above, I think this should be:

```
"prettier --write \"**/*.{js,jsx,ts,tsx,json,css,scss}\""
```

i.e. without the extra `*/` at start of the pattern?